tests/testenvironment.proj

```diff
@@ -48,6 +50,8 @@
   <ItemDefinitionGroup>
     <TestEnvironment>
       <TieredCompilation>0</TieredCompilation>
+      <DbgEnableMiniDump>1</DbgEnableMiniDump> <!-- Enable minidumps for all scenarios -->
+      <DbgMiniDumpName Condition="'$(TargetsWindows)' != 'true'">$HELIX_WORKITEM_UPLOAD_ROOT/coredump.%d.dmp</DbgMiniDumpName>
```

Comment:
Is there a reason we would want a different name format for each OS?

Reply:
The name format used an environment variable so we need to make it conditional on OS to get the syntax correct.

Reply:
Also, I haven't yet enabled crash-dump-on-assert for windows (where this would be used) so I haven't set it yet.